test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -63,6 +64,24 @@ call_log_manager::process_call_log()
         process_call_log_entry(call_log_entry);
 }
 
+session_simulator* 
+call_log_manager::get_session(json call_log_entry){
+    const std::string session_id = call_log_entry["session_id"].get<std::string>();
+
+    /*
+     * We should not perform an operation on a session with an ID that does not exist in the
+     * session map.
+     */
+    if (_session_map.find(session_id) == _session_map.end()) {
+        std::cerr << "Could not perform operation, session does not exist (session ID: "
+                    << session_id << ")" << std::endl;
+        return (nullptr);
```

Comment:
Not sure if we should return null or throw an error here?

Reply:
I believe this is something you should improve when you will do the error handling